tensorrt_llm/_torch/utils.py

```
@@ -283,3 +285,19 @@
 
 def get_per_request_piecewise_cuda_graph_flag() -> bool:
     return getattr(_global_attrs, 'per_request_piecewise_cuda_graph_flag', True)
+
+
+def create_lm_head_tp_mapping(mapping: Mapping) -> Mapping:
+    lm_head_tp_size = int(os.getenv('LM_HEAD_TP_SIZE', 2))
+    assert mapping.tp_size % lm_head_tp_size == 0
```

Ruff (0.12.2)

Use of `assert` detected

(S101)